jsonmerge_utils.py

```diff
@@ -5,10 +5,10 @@
 @author: GRENTOR
 """
 import os
+import logging
 #from tqdm import tqdm
 from merge_files import Merge
-
-
+logger = logging.getLogger("jsonmerge-logger")

```

Comment:
```suggestion
logger = logging.getLogger(__name__)
```
This ensures a more clean way of presenting logs imo.

Reply:
I was not aware of this! thank you I will have this done in an hour.